packages/wrangler/templates/test-new-worker.js

```diff
@@ -1,4 +1,4 @@
-import { unstable_dev } from "wrangler";
+const { unstable_dev } = require("wrangler");
```

Comment:
Why can't this be an ESM import?

Reply:
Setting up jest for ESM can be a follow-up task

Reply:
Does naming it with mjs instead of js work?

Reply:
It's not trivial to set-up: https://stackoverflow.com/a/69059786/5175932

Reply:
Bare minimal setup requires adding `"type": "module",` to package.json, installing `cross-env`, and setting the test script to `cross-env NODE_OPTIONS=--experimental-vm-modules jest`